blake2/src/consts.rs

```diff
@@ -14,10 +14,10 @@ pub static SIGMA : [[usize; 16]; 12] = [
 ];
 
 pub static BLAKE2B_IV : [u64; 8] = [
-    0x6a09e667f3bcc908, 0xbb67ae8584caa73b,
-    0x3c6ef372fe94f82b, 0xa54ff53a5f1d36f1,
-    0x510e527fade682d1, 0x9b05688c2b3e6c1f,
-    0x1f83d9abfb41bd6b, 0x5be0cd19137e2179,
```

Comment:
Constants are usually taken directly from the specifications and not intended to be read by humans, so I am not sure if its worth to introduce such change. @tarcieri what do you think?

Reply:
I didn't realize this was the case until later on when I added the explicit allow in constant files. I can go back and do that here.